### ISSUE TITLE
a11y issue (DAC-USER-SRJAWS-T1-12)

### ISSUE THREAD

**Issue by lc512k** [Financial-Times/next-front-page#1396]
A11y audits results revealed the following high-priority issue with the video carousel
<img width="1113" alt="screen shot 2016-12-10 at 12 54 10" src="https://cloud.githubusercontent.com/assets/3425322/21073510/2d6a462a-bed8-11e6-9b21-d0e641e7870c.png">

**Comment by lc512k** [comment]
@ironsidevsquincy, could be happening in the new video hub too?

**Comment by ironsidevsquincy** [comment]
There are no videos on the new video hub, but videos in articles could be affected. Though I thought we were just using the native player everywhere, so seems a bit odd

**Comment by aintgoin2goa** [comment]
Yeah o-video just uses the native player - this could be a browser bug?  We haven't added any keyboard controls that I'm aware of

**Comment by i-like-robots** [comment]
Can confirm, it's just the native player.
